test/e2e/run_volume_test.go

```diff
@@ -953,4 +953,19 @@ USER testuser`, fedoraMinimal)
 		Expect(volMount).Should(Exit(0))
 		Expect(volMount.OutputToString()).To(Equal("1000:1000"))
 	})
+
+	It("podman run -v with a relative dir", func() {
+		mountPath := filepath.Join(podmanTest.TempDir, "vol")
```

Comment:
Could you create a randomly named file in the vol path and then run a container and make sure the file exists at the destination path inside of container